Bottom: 3d1b70c31ba8a1c446dec7a3e57d1855fdf25d2d
Top:    cda5866557caa4ff0b1c3aeb31da46bd11070aeb
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:21:35 -0500

Refresh of zarlink

---

diff --git a/p.sh b/p.sh
new file mode 100755
index 0000000..048957b
--- /dev/null
+++ b/p.sh
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+stg refresh
+git branch -D ravi1
+stg publish ravi1
+stg puglish ravi
+git push github --all --force
+
